.travis.yml

```diff
@@ -6,6 +6,7 @@ stages:
   - cov
 
 node_js:
+  - '10'
   - '12'
```

Comment:
may as well add 14 while you're in here eh? I quite like adding in `current` and `lts/*` too just to make it a bit more future-proof